jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
+short-title: "Using the CircleCI Local CLI"
+description: "How to run local jobs with the CLI"
+categories: [troubleshooting]
+order: 10
+alias: /docs/2.0/local-jobs/
+---
+
+*[Build Environments]({{ site.baseurl }}/2.0/build/) > {{ page.title }}*
+
+* TOC
+{:toc}
+
+## Overview
+
+The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase. 
+
+You can also run `circleci` commands in your `config.yml` file for jobs that use the primary container image. This is particularly useful for globbing or splitting tests among containers. 
+
+**Note:** The Local CLI does not yet support Workflows.
+
+## Installing
+
+### Ubuntu 16.04+, Fedora 27+, and other Distros Supporting Snap Packages
+
+```
+sudo snap install docker circleci
+sudo snap connect circleci:docker docker
+```
```

Comment:
Do we really want to present commands without any context or introduction?

Reply:
What do you mean by this? That's what do we with the cURL instructions.

If you'd like a paragraph explaining what this is doing, I'd be more than happy to do that in another PR today or tomorrow. I figured you'd want me to be as succinct as possible.

Reply:
I like things being short, but in this case, these commands are the first things I saw after a title. Even just a sentence summarizing what this is doing would suffice.

Reply:
Addressed.